src/subscriber.ts

```diff
@@ -46,33 +47,91 @@ export interface PullResponse {
   receivedMessages: ReceivedMessage[];
 }
 
+/**
+ * Date object with nanosecond precision. Supports all standard Date arguments
+ * in addition to several custom types.
+ *
+ * @external PreciseDate
+ * @see {@link https://github.com/googleapis/nodejs-precise-date|PreciseDate}
+ */
 /**
  * Message objects provide a simple interface for users to get message data and
  * acknowledge the message.
  *
- * @private
- * @class
- *
- * @param {Subscriber} sub The parent subscriber.
- * @param {object} message The raw message response.
+ * @example
+ * subscription.on('message', message => {
+ *   // {
+ *   //   ackId: 'RUFeQBJMJAxESVMrQwsqWBFOBCEhPjA',
+ *   //   attributes: {key: 'value'},
+ *   //   data: Buffer.from('Hello, world!),
+ *   //   id: '1551297743043',
+ *   //   publishTime: new PreciseDate('2019-02-27T20:02:19.029534186Z'),
+ *   //   received: 1551297743043
```

Comment:
Really silly, but this line in a comment is missing a trailing comma.

Reply:
Oof, nice catch!